packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -567,18 +572,22 @@ export const samplerOperations = {
                 ),
             ),
         );
-
+        const samplerOps = subOps.filter(op => op.source !== ERC20BridgeSource.Balancer);
+        const nonSamplerOps = subOps.filter(op => op.source === ERC20BridgeSource.Balancer);
```

Comment:
Is this change just to reduce the load on the sampler contract?

Reply:
the sampler was reverting because `batchCall` didn't know what to do with empty bytes. we could change `batchCall` instead to skip over if `callDatas[i] == 0`, but 🤷